src/Engine/RadiumEngine.hpp

```diff
@@ -171,6 +171,83 @@ class RA_ENGINE_API RadiumEngine
      */
     Core::Aabb computeSceneAabb() const;
 
+    /// \name Time Management.
+    /// {
+
+    /**
+     * Toggles effective time flow or constant time flow.
+     */
+    void setRealTime( bool realTime );
+
+    /**
+     * Sets the time delta between two frames for constant time flow.
+     */
+    void setConstantTimeFlowDt( Scalar dt );
+
+    /**
+     * Toggles on/off time flow.
+     */
+    void play( bool isPlaying );
+
+    /**
+     * Toggles time flow for one frame.
+     */
+    void step();
+
+    /**
+     * Resets time to 0, same as \code goTo( uint( 0 ) ); \endCode.
+     */
+    void reset();
+
+    /**
+     * Sets the `start` time for the time window.
+     * \note A negative start time is considered as 0.
+     */
+    void setStartTime( Scalar t );
+
+    /**
+     * Sets the `start` time for the time window.
+     */
+    Scalar getStartTime() const;
+
+    /**
+     * Sets the `end` time for the time window.
+     * \note A negative end time makes the time flow indefinitely.
+     */
+    void setEndTime( Scalar t );
+
+    /**
+     * \Returns the `end` time for the time window.
+     */
+    Scalar getEndTime() const;
+
+    /**
+     * Toggles the PingPong mode.
```

Comment:
What is "the PingPong" mode ? 
If I understood well the code, and even if longer, I suggest to use instead ForwardBackward